### DISCUSSION TITLE
Questions about two bugs in the cachebench log file

### DISCUSSION THREAD

**Discussion by wangyuyue** [facebook/CacheLib#229] [Q&A]
Hi developers,
when I run the cachebench and check the log files, I notice two bugs frequently occur.
First, the eviction success rate in `Evict Attempts: 7,105,134 Success: -259625561934544.84%` is usually a negative number. So I guess the calculation is not correct.
Second, sometimes the log has missing timestamps. For example, the timestamps are 11:45:18 and 11:46:18, and then directly jump to 11:52:18. This bug is rare, but It would be great to be resolved, as a contiguous log makes performance analysis easier.

Thanks a lot.

**Comment by haowu14** [comment]
Hi, can you share a cachebench config that generates the negative success rate?

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
